### ISSUE TITLE
tasks/tasks/build_image.yml not found

### ISSUE THREAD

**Issue by MikeSpreitzer** [containercafe/containercafe#125]
With ansible 1.9.5:
```
mjs10:containercafe mspreitz$ ( cd ansible; \
>   ansible-playbook -v -i ../examples/envs/dev-vbox/radiant01.hosts env-basics.yml \
>       -e "envs=../examples/envs env_name=dev-vbox" )
ERROR: file could not read: /Users/mspreitz/Documents/project-alchemy/containercafe/ansible/roles/proxy/tasks/tasks/build_image.yml
```

**Comment by MikeSpreitzer** [comment]
Changing the include statement from
```
    include: tasks/build_image.yml
```
to
```
    include: build_image.yml
```
fixes the problem for Ansible 1.9.5.

**Comment by MikeSpreitzer** [comment]
That fixes the problem in Ansible 2.1.1.0 too.  I am going to submit a PR making this change.

**Comment by MikeSpreitzer** [comment]
CC @akanso
